addons/point_of_sale/models/pos_order.py

```diff
@@ -294,6 +295,7 @@ def _get_pos_anglo_saxon_price_unit(self, product, partner_id, quantity):
     refunded_order_ids = fields.Many2many('pos.order', compute='_compute_refund_related_fields')
     has_refundable_lines = fields.Boolean('Has Refundable Lines', compute='_compute_has_refundable_lines')
     refunded_orders_count = fields.Integer(compute='_compute_refund_related_fields')
+    ticket_code = fields.Char(help='5 digits alphanumeric code to be used by portal user to request an invoice')
```

Comment:
This should also be sent back to the front end when we want to reprint an older receipt. Check how it is done for `access_token` in `_export_for_ui`.
This also needs to be sent in the pos_restaurant module when fetching draft orders. Add it in `_get_fields_for_draft_order`